src/examples/dropdowns/SelectValidation.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import {
+  Dropdown,
+  Field,
+  Menu,
+  Item,
+  Select,
+  Label,
+  Message
+} from '@zendeskgarden/react-dropdowns';
+
+const StyledSelect = styled(Select)`
+  * {
+    box-sizing: initial;
+  }
+`;
+
+const StyledField = styled(Field)`
+  * {
+    /* stylelint-disable-next-line declaration-no-important */
+    appearance: none !important;
```

Comment:
It seems like some CSS from bedrock is causing the styles to render this in Safari:

<img width="823" alt="Screen Shot 2020-06-01 at 10 13 32 AM" src="https://user-images.githubusercontent.com/1811365/83435144-94abff80-a3f0-11ea-8744-295c85fc0cf3.png">

Targeting the faux input with `appearance: none !important;` results in the following:

<img width="823" alt="Screen Shot 2020-06-01 at 10 14 41 AM" src="https://user-images.githubusercontent.com/1811365/83435270-c329da80-a3f0-11ea-8351-30af268a5ce9.png">

Having a `StyledField` with a `!important` to target an element is probably not ideal. Any suggestions here?

Reply:
Let's remove all the `StyledField` overrides and address this in `react-components` :wink: